src/Inkluzitron/Program.cs

```diff
@@ -89,6 +89,8 @@ static private async Task MainAsync(string[] args)
                     {
                         if (category.StartsWith("Microsoft.EntityFrameworkCore"))
                             return category == "Microsoft.EntityFrameworkCore.Migrations" || level > LogLevel.Information;
+                        else if (category.StartsWith("System.Net.Http.HttpClient.Default"))
```

Comment:
To logování na externí služby pomocí HttpClient bych nechal. Pak půjde z logu jednoduše zjistit co se děje.